app/pulseapi/creators/models.py

```diff
@@ -28,7 +28,7 @@ class Creator(models.Model):
     """
     name = models.CharField(max_length=140)
     user = models.ForeignKey(
-        UserProfile,
+        settings.AUTH_USER_MODEL,
```

Comment:
I'm...not sure what I was getting at here either. Basically the Auth user model is the central identity all other models that are representations of people connect to, correct? And through that we can reach all other versions of those models?

For example, I submit a project to Pulse, and list you as a creator (tied to your email address). Then we both have profiles on the "people" page. Can we easily show that project on our "people" page profiles?